src/catalog/factories/wall-factory-3d.js

```diff
@@ -5,11 +5,12 @@ import {
   Vector2,
   BoxGeometry,
   MeshBasicMaterial,
+  MeshPhongMaterial,
   Group
 } from 'three';
 
 import ThreeBSP from '../../utils/threeCSG.es6';
-import {verticesDistance} from '../../utils/geometry';
+import { verticesDistance } from '../../utils/geometry';
 import * as SharedStyle from '../../shared-style';
 
 const halfPI = Math.PI / 2;
```

Comment:
Not really necessary as you could define your own wall in the catalog

Reply:
this was just auto formatting I think, from saving the file again. I didn't change anything here.